src/pages/Login/login.test.jsx

```diff
@@ -33,3 +40,71 @@ describe('Login Page Unitary Testing', () => {
     expect(screen.getByText('Campo Obrigatório')).toBeTruthy()
   })
 })
+
+describe('Login Page Integration Testing', () => {
+  test('Should fail to sign in', async () => {
+    apiMock.onPost('/user/login').reply(400, {})
+
+    // Silencia console.log temporariamente
+    const consoleLogMock = vi.spyOn(console, 'log').mockImplementation(() => {})
+
+    render(
+      <>
+        <Login />
+        <ToastContainer />
+      </>,
+    )
+
+    const emailField = screen.getByPlaceholderText('E-mail')
+    const passwordField = screen.getByPlaceholderText('Senha')
+
+    await userEvent.type(emailField, 'teste@mail.com')
+    await userEvent.type(passwordField, 'Void@@68465')
+
+    const button = screen.getByText('Entrar')
+    await userEvent.click(button)
+
+    expect(
+      screen.getByText('Erro na autenticação, verifique seu e-mail ou senha'),
+    ).toBeTruthy()
+    expect(
+      screen.getByText('Erro na autenticação, verifique seu e-mail ou senha'),
+    ).toBeInTheDocument()
```

Comment:
podemos usar o `toBeVisible()`, assim garantimos que o elemento está de fato visível em tela

```suggestion
       expect(
      screen.getByText('Erro na autenticação, verifique seu e-mail ou senha'),
    ).toBeVisible()

```

Reply:
Boa, nunca tinha usado, achei legal adicionar ele no teste ao invés de substituir